pygmt/src/surface.py

```diff
@@ -17,9 +17,14 @@
 
 @fmt_docstring
 @use_alias(
+    C="convergence",
+    G="outgrid",
     I="spacing",
+    Ll="lower",
+    Lu="upper",
+    M="max_radius",
```

Comment:
Based on this convention in the Contributors Guide (please see section [Code style](https://www.pygmt.org/dev/contributing.html#code-style)), we should use `maxradius` (no underscore) instead of `max_radius`:
> When wrapping a new alias, use an underscore to separate words bridged by vowels (aeiou), such as no_skip and z_only. Do not use an underscore to separate words bridged only by consonants, such as distcalc, and crossprofile. This convention is not applied by the code checking tools, but the PyGMT maintainers will comment on any pull requests as needed.
```suggestion
    M="maxradius",
```